db/traits/ActiveRecordAccessTrait.php

```diff
@@ -288,4 +290,18 @@ private function addAccessError($action, $attribute)
         $this->addError($attribute, $msg);
         \Yii::info('User ID: #' . \Yii::$app->user->id . ' | ' . $msg, get_called_class());
     }
+    
+    /**
+     * Return correct part of check in set  query for current DB
+     * @param $accessRead
+     * @param $authItems
+     * @return string
+     */
+    private static function getInSetQueryPart($accessRead, $authItems)
+    {
+        if (Yii::$app->db->getDriverName() === 'mysql') {
+            return 'FIND_IN_SET(' . $accessRead . ', "' . $authItems . '") > 0';
+        }
+        return " '" . $accessRead . "'= SOME (string_to_array('$authItems', ','))";
```

Comment:
Isn't this PgSQL specific.
What's about a switch statement for the supported DBs, including an exception for non-supported ones?

CC: @StalkAlex @Quexer69